tools/make-config.py

```diff
@@ -1103,12 +1103,12 @@ def make_config(ovl_name: str, version: str):
         hydrate_psp_symbols(splat_config_path, splat_config, version)
     else:
         found = hydrate_psx_duplicate_symbols(splat_config, ovl_name, version)
-        if found > 0:
+        if found is not None and found > 0:
```

Comment:
I think this `None` is coming from https://github.com/JoshSchreuder/sotn-decomp/blob/6c28f88ec14843f16c00aeb3a55ca4ef1db14743/tools/make-config.py#L980 ?

Would you prefer if I update that to return 0 instead?